src/test/regress/expected/multi_router_planner.out

```diff
@@ -1797,7 +1797,7 @@ DROP MATERIALIZED VIEW mv_articles_hash;
 DEBUG:  drop auto-cascades to type mv_articles_hash
 DEBUG:  drop auto-cascades to type mv_articles_hash[]
 DEBUG:  drop auto-cascades to rule _RETURN on materialized view mv_articles_hash
-DEBUG:  EventTriggerInvoke 16761
+DEBUG:  EventTriggerInvoke 16767
```

Comment:
But there's no need to have the log level to DEBUG here, afaics. So it should be easy to get rid of.